.prow.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+CSI_PROW_TESTS_SANITY="sanity"
```

Comment:
The Prow job that you added will all tests that are enabled in `.prow.sh`. This includes E2E and sanity testing, but that fails because more work is needed (e.g. defining how to install the driver).

You need to limit testing to simple build and unit testing with:
```
diff --git a/.prow.sh b/.prow.sh
index d5f9cda..cb64ff0 100755
--- a/.prow.sh
+++ b/.prow.sh
@@ -1,6 +1,8 @@
 #! /bin/bash
 
-CSI_PROW_TESTS_SANITY="sanity"
+# Only these tests work for csi-drivers-flex, E2E and sanity testing
+# will require further work.
+: ${CSI_PROW_TESTS:="unit"}
 
 . release-tools/prow.sh
 
```

Reply:
Updated.